graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/predicate/FieldEvaluator.java

```diff
@@ -240,22 +229,12 @@ public Object getFieldValue(Object source, String fieldName) {
                 .executionId(environment.getExecutionId())
                 .fragmentsByName(environment.getFragmentsByName())
                 .graphQLSchema(environment.getGraphQLSchema())
+                .arguments(environment.getArguments())
```

Comment:
I don't know if this is a valid substitute to `valuesResolver.getArgumentValues` but don't know if there's a way to get to a `Field` object from `SelectedField` or maybe `DataFetchingFieldSelectionSet`